### ISSUE TITLE
A name for the snap

### ISSUE THREAD

**Issue by soumyaDghosh** [amit9838/weather#7]
Hiii, can you please suggest a name for your app which I will use in the snap-store for listing? The word weather is a vetted name, it's not allowed to use that. So, anything else?

**Comment by amit9838** [comment]
How about weather-snap?
This will indicate snapped version of weather without loosing actual identity 'Weather'

**Comment by soumyaDghosh** [comment]
Such names are actually also discouraged. But.. what I am thinking is requesting the name. But, I would request you also to think a new name, as that's a very common one. How about, `abohaowa` or `mousam`?

**Comment by amit9838** [comment]
These names are good for Indian perspective but, since we are building this for global audience people may face difficulty in remembering these name. I think you understood what I'm trying to say.

**Comment by soumyaDghosh** [comment]
> These names are good for Indian perspective but, since we are building this for global audience people may face difficulty in remembering these name. I think you understood what I'm trying to say.

But, if you look into gnome HIG, these words have a meaning. Also, I don't think `mousam` isn't a big word to remember. Also, regarding global audience, how the app looks and works matters I guess. Names like `bavarder`, `denaro` aren't also English words. Still they are working pretty fine. Sorry, for my different perspective in this regards.

**Comment by amit9838** [comment]
'Mausam' seems fine for me. I think a better idea would be to take more suggestions.
Is there any way to create pool on GitHub?

**Comment by soumyaDghosh** [comment]
No, github doesn't have such option to create poll. But, you can have an issue, pinned at top.

**Comment by soumyaDghosh** [comment]
like here:
https://github.com/NickvisionApps/TubeConverter/issues/351

**Comment by amit9838** [comment]
![IMG_20230618_000313_511.jpg](https://github.com/amit9838/weather/assets/61614402/8b4ca340-c92a-494f-87b4-9896fa64e43d)

Here are some good names

**Comment by soumyaDghosh** [comment]
SnapStorms? Or, Sunshade?

**Comment by amit9838** [comment]
> SnapStorms? Or, Sunshade?

Let's take some more inputs

**Comment by amit9838** [comment]
Hi @soumyaDghosh have you got any new names? or we should proceed with any name among these.

**Comment by amit9838** [comment]
Ok so, lets proceed with `mousam` unitll we find any strong reason to change the name. Btw I also kind of like this name :slightly_smiling_face: .

**Comment by amit9838** [comment]
How are you going to maintain compilation and updates, like - manually or bot?

**Comment by soumyaDghosh** [comment]
> Btw I also kind of like this name slightly_smiling_face .

Afterall we're the people of same mind na!



> How are you going to maintain compilation and updates, like - manually or bot?

If you allow me to be a contributor to this repo, I will add the snap manifest directly to this repo, then, connect this with the snapcraft website, would like to have you there, directly under your account and me as a collaborator or vice-e-versa. And with every updates you push, just update the source-tag. It's more easy actually than flatpak.

**Comment by amit9838** [comment]
ok ,i got the idea, so can you build from latest release tag. and raise the pr so i can directly at you as collabortor.

**Comment by soumyaDghosh** [comment]
> ok ,i got the idea, so can you build from latest release tag. and raise the pr so i can directly at you as collabortor.

I can build from any `tag`, `commit`, `branch` or directly from the latest commit

**Comment by amit9838** [comment]
It would be great if you build from latest release tag so that both platforms stays parallel.

**Comment by amit9838** [comment]
You can see more info about release tag from `manifest file`

**Comment by soumyaDghosh** [comment]
> It would be great if you build from latest release tag so that both platforms stays parallel.

I would.

**Comment by amit9838** [comment]
can you send me a mail here amitchaudhary0539@gmail.com ?
